db/migrate/20230927012615_drop_billing_locks.rb

```diff
@@ -0,0 +1,14 @@
+class DropBillingLocks < ActiveRecord::Migration[5.2]
+  def up
+    drop_table :billing_locks
+  end
+
+  def down
+    table_opts = System::Database.mysql? ? { options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin" } : {}
+    datetime_opts = System::Database.oracle? ? { precision: 6 } : {}
+
+    create_table "billing_locks", primary_key: "account_id", force: :cascade, **table_opts do |t|
+      t.datetime "created_at", null: false, **datetime_opts
+    end
+  end
```

Comment:
I'm not sure it makes sense to have a `#down` method. If the user rolls back, the table would be empty anyway because the `BillingLock` model is removed.

Reply:
Having the DB empty is OK.
In fact, it is expected that the DB remains empty after billing has completed, because the records were expected to be deleted after the block completed: https://github.com/3scale/porta/pull/3403/files#diff-fe08cb8a1ec2dece23621f428b7abc88974b06ac2fbfb6e845817913b5dcfb96L70

Reply:
Also this table is unused since previous changes. So it is just a useless table now. In general we don't need these `down` methods much. Mostly convenient while developing as you earlier stated.